kurobako/solver/optuna.py

```diff
@@ -180,8 +181,8 @@ def tell(self, evaluated_trial: solver.EvaluatedTrial):
 
         assert current_step <= self._problem.last_step
         if self._problem.last_step == current_step:
-            self._study.tell(trial, values=values)
-            self._study._log_completed_trial(trial, values)
+            frozen_trial = self._study.tell(trial, values=values)
+            self._study._log_completed_trial(frozen_trial)
```

Comment:
The problem here was addressed in https://github.com/optuna/kurobako-py/pull/19, and the PR has been merged. So, please sync with the master branch 🙏